modules/retiolum/default.nix

```diff
@@ -32,6 +32,7 @@ in {
     services.tinc.networks.${netname} = {
       name = cfg.nodename;
       extraConfig = ''
+        Compression = 11  # best lzo
```

Comment:
Is this higher or faster compression than the default?

Reply:
The default is off I think

https://www.tinc-vpn.org/documentation/tinc.conf.5

```
     Compression = level (0)
             This option sets the level of compression used for UDP packets.  Possible values are 0 (off), 1 (fast zlib) and any
             integer up to 9 (best zlib), 10 (fast lzo) and 11 (best lzo).
```